armi/reactor/converters/uniformMesh.py

```diff
@@ -115,7 +115,9 @@ def convert(self, r=None):
         self._setParamsToUpdate()
         self._computeAverageAxialMesh()
         self._buildAllUniformAssemblies()
-        self._mapStateFromReactorToOther(self._sourceReactor, self.convReactor)
+        self._mapStateFromReactorToOther(
+            self._sourceReactor, self.convReactor, mapNumberDensities=False
```

Comment:
We need `mapNumberDensities=False` here in a `self.options.nonUniformAssems` case. But I'm not sure it will be right for `self.options.detailedAxialExpansion`. I will need to test this.